doc: clarify fixes and refs trailer guidance (#64421)
nodejs/node: doc/contributing/pull-requests.md

3. Wrap all other lines at 72 columns (except for long URLs).

4. If your patch fixes an open issue, include a reference in the pull request
   description. Use the `Fixes:` prefix and the full issue URL. For other
   references, use `Refs:`.

   `Fixes:` and `Refs:` trailers are automatically added to the commit message
   when the pull request lands. If the pull request lands as several commits,
   the trailers from the description are added to each commit by default. When

>    landing the pull request manually, add the trailers to the commit message.

Reviewer: I don't understand this, I find it confusing, can you clarify or remove this? "landing manually" could mean several things, but no one should be hand-crafting commit messages that end up on `main`, they should be generated by NCU – and NCU adds the trailers if found in the PR description.
```suggestion
   the trailers from the description are added to each commit by default.
```

Author: Removed in [`7c8d9d8` (this PR)](https://github.com/nodejs/node/pull/64421/changes/7c8d9d880beec67888b7fc485cdaf6d4b5499f35)
